README.md

```diff
@@ -154,6 +154,44 @@ To track deep links in iOS you must add the following to your `AppDelegate.m` fi
   return YES;
 }
 ```
+### Native Anonymous ID 
+
+If you need to generate an `anonymousId` either natively or before the Analytics React Native package is initialized, you can send that value across the bridge and utilize it in the JavaScript layer. Segment does not generate this value for you, but does extend the functionality necessary to dispatch the value across the bridge. For reference, you can find a working example in the app and reference the code below: 
+
+**iOS**
+```objc
+...
+#import <segment_analytics_react_native-Swift.h>
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  ...
+  // generate your anonymousId value
+  // dispatch it across the bridge
+
+  [AnalyticsReactNative setAnonymousId: @"My-New-Native-Id"];
+  return yes
+}
+```
+**Android**
+```java
+// MainApplication.java
+...
+import com.segmentanalyticsreactnative.AnalyticsReactNativePackage;
+
+...
+  @Override
+  public void onCreate() {
+    super.onCreate();
+    ...
+
+  // generate your anonymousId value
+  // dispatch it across the bridge
+
+  analytics.setAnonymousId("My-New-Native-Id");
```

Comment:
You need to show here that in Android you need to manually add your package so that you have the `analytics` reference.